### DISCUSSION TITLE
Sayfa oluşturmak

### DISCUSSION THREAD

**Discussion by EchoBilisim** [realsuayip/django-sozluk#77] [Ideas]
Merhaba sistem oldukça şık ve kullanışlı sistem için bir kaç önerim olacaktır.

Sayfa oluşturma seçeneği eklerseniz güzel olacaktır örnek hakkımızda gizlilik politikası gibi sayfalar için sayfa oluşturma seçeneği olursa oldukça iyi olacaktır .

Yan blog oluşturma yanda var olan bir blog var içinde iletişim gibi seçenekler var içinde onun altında blog oluşturma ve oraya açılmış olan sayfalar eklenir veya link bölümler kategoriler eklenir bu tür bir seçenek kullanış açısından oldukça iyi olacaktır.

Mobilde entry oluştur seçeneği var ancak pc den girince entry oluştur seçeneği yok onun için pc için üst menüde olay mesajlar seçeneğin yanında entry giriş butonu eklenmesi oldukça kullanışlı olacaktır entry eklemek isteyen yeni sistemle tanışanlar için eklemek kullanmak daha basit olacaktır.

Karanlık tema seçme seçeneğini üst menüye eklenmesi pc den girince sıkıntı değil yanda oluyor genelde ama mobilden girince sayfanın en altında oluyor onuda üst menüye kısa yol olarak eklenirse mobil dede üst menüde olur ordan hızlı bir şekilde değiştirebilir kullanım açısından oldukça iyi olur.

**Comment by realsuayip** [comment]
Sayfa oluşturma seçeneği var, admin panelinden düz sayfa ekleme kısmından ekleyebilirsiniz. Örnek: 
https://sozluk.me/faq/

Yan blog'dan kastınız tam nedir bilmiyorum ama duyurular özelliğini blog şeklinde kullanabilirsiniz.

Entry ekleme seçeneği masaüstünde'de var fakat oraya yönlendiren bir bağlantı yok, dilerseniz ekleyebilirsiniz. yine karanlık mod için html dosyalarında benzer bir değişiklik yapabilirsiniz.
